source/NVDAObjects/window/winword.py

```diff
@@ -426,8 +426,16 @@ def iterate(self):
 			if self.direction=="previous":
 				index=itemCount-(index-1)
 			collectionItem=items[index]
-			item=self.quickNavItemClass(self.itemType,self.document,collectionItem)
-			itemRange=item.rangeObj
+			try:
+				item=self.quickNavItemClass(self.itemType,self.document,collectionItem)
+				itemRange=item.rangeObj
+			except:

```

Comment:
I assume it raises COMError? If so perhaps best to just catch that specifically, so as to not hide other more critical errors.